### PR TITLE
Fix intermittent StaleElementReferenceException from set_region()

### PR DESCRIPTION
**Pull request by bobsilverberg**
Here's another one, @davehunt, related to a Fireplace testing failure.  You can see the failure at [1], and I've got a Travis job running against this branch at [2] (to see if it removes the failure).

[1] https://fireplace-tests.s3.amazonaws.com/bobsilverberg/fireplace/33/33.3/index.html.
[2] https://travis-ci.org/bobsilverberg/fireplace/jobs/72315081

### CODE REVIEW

File: pages/desktop/regions/debug.py [hunk 1]
```diff
@@ -12,10 +12,12 @@
...
+    _region_option_locator = (By.CSS_SELECTOR, 'select[name="region"] option[value="%s"]')
 
     def __init__(self, testsetup):
         Page.__init__(self, testsetup)
         self.wait_for_element_visible(*self._region_select_locator)
 
     def select_region(self, region):
-        self.select_option(region, self._region_select_locator)
+        self.selenium.find_element(self._region_option_locator[0],
+                                   self._region_option_locator[1] % region).click()
```
Comment (davehunt) [inline pages/desktop/regions/debug.py]: Can we just use `Select(self.selenium.find_element(*_region_select_locator)).select_by_value(region)` here? See [Select.select_by_value](http://selenium-python.readthedocs.org/en/latest/api.html?highlight=script#selenium.webdriver.support.select.Select.select_by_value).

Comment (bobsilverberg) [inline pages/desktop/regions/debug.py]: @davehunt Yes, that's way better. Thanks for the suggestion!

**Comment by bobsilverberg** [comment]
@davehunt I updated the PR with your suggestion. r?

File: pages/desktop/regions/debug.py [hunk 2]
```diff
@@ -18,4 +19,6 @@ def __init__(self, testsetup):
         self.wait_for_element_visible(*self._region_select_locator)
 
     def select_region(self, region):
-        self.select_option(region, self._region_select_locator)
+        Select(
+            self.selenium.find_element(
+                *self._region_select_locator)).select_by_value(region)
```
Comment (davehunt) [inline pages/desktop/regions/debug.py]: I'm not keen on the style here, but r+ if this is working (I haven't run the tests, let me know if you would like me to - or perhaps trigger an adhoc job on staging)

**Comment by bobsilverberg** [comment]
You and your style nits, @davehunt!  I tested it locally, but just to be sure I kicked off an adhoc at http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/marketplace.adhoc/16/. If all is well I'll squash and merge.

**Comment by bobsilverberg** [comment]
There was one error in that adhoc run, but it is unrelated. Merging.

**Pull request merged**